projects/ad9371/profiles/tx_bw100_orx_bw100_rx_bw100/myk_ad9528init.c

```diff
@@ -0,0 +1,61 @@
+/* AD9528 data structure initialization file */
+#include <stdint.h>
+#include "common.h"
+#include "t_ad9528.h"
+
+static spiSettings_t clockSpiSettings = {
+	1, //chip select Index

```

Comment:
i know there was a discussion about designated initializer. Should we start using it ?

Reply:
These files are generated by TES GUI, so these kind of changes should be implemented there.

Reply:
ack.